src/import-data/ImportDataOverview.ts

```diff
@@ -56,23 +53,13 @@ export const ImportDataOverview = (props: ImportDataOverviewProps): ReactNode =>
 
   return div({ style: styles.card }, [
     h2({ style: styles.title }, [getTitleForImportRequest(importRequest)]),
-    'url' in importRequest && div({ style: { fontSize: 16 } }, ['From: ', importRequest.url.hostname]),
+    h4({ style: { fontSize: 16 } }, ['Dataset source:']),
```

Comment:
Heading levels should not be skipped. So the heading after an h2 should be an h3, not an h4.

https://webaim.org/techniques/semanticstructure/#headings